config/example.yaml

```diff
@@ -0,0 +1,18 @@
+server:
+  endpointUri: ${ENDPOINTURI}
+  basePath: /api/v1
```

Comment:
I know this is how our dropwizard APIs do this, but I think the version "v1" should not go in the configuration file. The version should be set somewhere in source control, because it refers to the specific commit/version of the API.

Reply:
Good point. I removed this from the configuration file and will get the version from the swagger file.